editor/src/components/canvas/controls/select-mode/select-mode.spec.browser2.tsx

```diff
@@ -386,7 +388,44 @@ describe('Select Mode Double Clicking', () => {
     expect(renderResult.getEditorState().editor.selectedViews).toEqual([desiredPath])
   })
 
-  it('Six double clicks will focus a generated Card and select its root element', async () => {
+  it('Five double clicks will focus a generated Card and select its root element', async () => {
+    // prettier-ignore
+    const desiredPath = EP.fromString(
+      'sb' +                 // Skipped as it's the storyboard
+      '/scene-CardList' +    // Skipped because we skip over Scenes
+      '/CardList-instance' + // <- First double click
+      ":38e" +               // <- Third double click skips fragment 
```

Comment:
Is it possible to create a hard coded uid for the fragments rather than relying on this generated one staying the same?

Reply:
Ah yes, there's a test function for this now IIRC.

Reply:
Putting this in the backlog, as it's turning into a non-trivial set of changes.